src/windows/rpi-imager.nsi.in

```diff
@@ -4,37 +4,38 @@
 #              by Vlasis K. Barkas aka Red Wine red_wine@freemail.gr Sep 2006               
 ############################################################################################
 
-!define APP_NAME "Raspberry Pi Imager"
-!define COMP_NAME "Raspberry Pi Ltd"
-!define VERSION "@IMAGER_VERSION_STR@"
-!define INSTALLER_NAME "imager-${VERSION}.exe"
-!define COPYRIGHT "Raspberry Pi Ltd"
-!define DESCRIPTION "Raspberry Pi Imager"
-!define MAIN_APP_EXE "rpi-imager.exe"
-!define INSTALL_TYPE "SetShellVarContext all"
-!define REG_ROOT "HKCU"
-!define REG_APP_PATH "Software\Microsoft\Windows\CurrentVersion\App Paths\${MAIN_APP_EXE}"
-!define UNINSTALL_PATH "Software\Microsoft\Windows\CurrentVersion\Uninstall\${APP_NAME}"
+!define APP_NAME           "Raspberry Pi Imager"
+!define COMP_NAME          "Raspberry Pi Ltd"
+!define VERSION            "@IMAGER_VERSION_STR@"
+!define INSTALLER_NAME     "imager-${VERSION}.exe"
+!define /date CURRENT_YEAR "%Y"
+!define COPYRIGHT          "${CURRENT_YEAR} ${COMP_NAME}"

```

Comment:
Just a small nit-pick, but as you've changed the white-space on every line, that makes it very hard to see where the actual-text has changed (e.g. for `COPYRIGHT`) vs. where only the white-space has changed (e.g. for `APP_NAME`).

Reply:
@lurch 

I aligned variables and values with space to have a beter view about values setup.
The original system is less clear then the current one.
Now you can see with a simple view what variable has what value.
This changes is one time for all.